plugins/spacelift/spacectl.go

```diff
@@ -0,0 +1,22 @@
+package spacelift
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func SpaceliftCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Spacelift CLI",
+		Runs:      []string{"spacectl"},
+		DocsURL:   sdk.URL("https://github.com/spacelift-io/spacectl"),
+		NeedsAuth: needsauth.NotForHelpOrVersion(),
```

Comment:
I noticed that `spacectl h` is a help command as well, so we can exclude 1Password authentication for that subcommand.

Reply:
Also, I think, we can exclude 1Password authentication for `spacectl profile` commands, [like how Fastly shell plugin does](https://github.com/1Password/shell-plugins/pull/169). Since those commands are for controlling the profile stored locally on the config file and not querying the Spacelift platform itself, disabling auth for that makes sense to me.